editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx

```diff
@@ -77,8 +80,34 @@ export interface NavigatorItemDragAndDropWrapperProps {
   visibleNavigatorTargets: Array<NavigatorEntry>
 }
 
-function canDrop(props: NavigatorItemDragAndDropWrapperProps, dropSource: ElementPath): boolean {
-  return !EP.isDescendantOfOrEqualTo(props.navigatorEntry.elementPath, dropSource)
+function notDescendant(
+  draggedOnto: NavigatorItemDragAndDropWrapperProps,
+  draggedItem: ElementPath,
+): boolean {
+  return !EP.isDescendantOfOrEqualTo(draggedOnto.navigatorEntry.elementPath, draggedItem)
+}
+
+function canDrop(
+  editorState: EditorState,
+  draggedItem: NavigatorItemDragAndDropWrapperProps,
+  draggedOnto: NavigatorItemDragAndDropWrapperProps,
+): boolean {
+  // TODO: What is this checking?
```

Comment:
This is checking that the drop target can accept children if the drag will result in a reparent onto that target

Reply:
Doh, this was a reminder to myself to check this!